src/component/plotly-dash-preview/render.js

```diff
@@ -14,16 +14,19 @@ const cst = require('./constants')
 function render (info, opts, sendToMain) {
   const result = {}
 
-  let createBrowserWindowOpts = info.browserSize ? info.browserSize : {}
-  createBrowserWindowOpts['show'] = opts.debug
+  let windowOpts = info.browserSize ? info.browserSize : { 'width': 800, 'height': 600 }
+  windowOpts['enableLargerThanScreen'] = true
+  windowOpts['useContentSize'] = true
+  windowOpts['show'] = opts.debug
 
-  let win = remote.createBrowserWindow(createBrowserWindowOpts)
+  let win = remote.createBrowserWindow(windowOpts)
   const contents = win.webContents
   const session = contents.session
 
   // Clear cookies before loading URL
   session.clearStorageData({}, () => {
     win.loadURL(info.url)
+    win.webContents.executeJavaScript(`window.resizeTo(${windowOpts.width}, ${windowOpts.height})`)
```

Comment:
Why is this necessary? It seems like you already set the window's size. :thinking:

Reply:
For some reason, it does not work when you do it with :```let win = remote.createBrowserWindow(createBrowserWindowOpts)```
It seems to be doing nothing.

I also tried `win.setSize` and `win.setContentSize` but they give me an error.

Reply:
Actually, I think it is because we need to round off window size as well:
![image](https://user-images.githubusercontent.com/1546532/65462951-d679b480-de24-11e9-993d-82e30bfce0bd.png)

I will try that and get back to you..